test/cypress/integration/simplified-account/service-settings/stripe-details/task-summary-test-helpers.js

```diff
@@ -0,0 +1,39 @@
+const taskStatus = {
+  NOT_STARTED: 'Not yet started',
+  COMPLETE: 'Completed',
+  CANNOT_START: 'Cannot start yet'
```

Comment:
Do we need to test for "In progress"?

Reply:
given that "in progress" is not implemented, we do not at this stage